edb/server/compiler/compiler.py

```diff
@@ -707,6 +722,19 @@ def compile_in_tx(
     ) -> Tuple[
         dbstate.QueryUnitGroup, Optional[dbstate.CompilerConnectionState]
     ]:
+        if request.input_language is enums.InputLanguage.SQL_PARAMS:
+            tx = state.current_tx()
+            return (
```

Comment:
This doesn't type check? Can we return a proper QueryUnitGroup instead?

Reply:
I fixed all type checking issues.